app/assets/javascripts/application-secret.js

```diff
@@ -0,0 +1,23 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
+// listed below.
+//
+// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
+// or vendor/assets/javascripts of plugins, if any, can be referenced here using a relative path.
+//
+// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
+// the compiled file.
+//
+// WARNING: THE FIRST BLANK LINE MARKS THE END OF WHAT'S TO BE PROCESSED, ANY BLANK LINE SHOULD
+// GO AFTER THE REQUIRES BELOW.
+//
+/* eslint-disable spaced-comment */
+//= require jquery
+//= require jquery-ui
+//= require jquery_ujs
+//= require action_cable
+//= require_directory ./lib
+//= require ./cloudcarousel-secret
+//= require ./metamaps.secret.bundle
+//= require ./Metamaps.ServerData
+//= require homepageVimeoFallback
+/* eslint-enable spaced-comment */
```

Comment:
Why are you using multiple bundles exactly? I'm not opposed, just interested

Maybe we could use this as an opportunity to implement a real AB testing framework. I've started researching this here: https://www.reddit.com/r/javascript/comments/3x2b56/ab_testing_and_single_page_application_good/. I was considering just using code splitting, but now I'm thinking about how we could set that up to be integrated like this with different bundles.